matrix_io/vision/v1/vision.proto

```diff
@@ -212,12 +215,13 @@ message VehicleConfig {
 }
 
 enum EventTag {
-  TRACKING_START = 0;
-  TRACKING_END = 1;
-  AREA_ENTER = 2;
-  AREA_EXIT = 3;
-  ZONE_ENTER = 4;
-  ZONE_EXIT = 5;
+  UNKNOWN = 1;
+  TRACKING_START = 2;
+  TRACKING_END = 3;
+  AREA_ENTER = 4;
+  AREA_EXIT = 5;
```

Comment:
Some of these might be hardcoded in vehicle/ves. I think @PatriciaReyes  should also take a look...

Reply:
Thanks @arhuaco 👍 for bringing these up. We seem to be using the constant name across all projects I checked so far. I'll run a test build of python, node and java packages and try to do some tests. Perhaps @hpsaturn could do an initial Android run ? 

We discussed this before with @campos537, so I'm counting on him to help us adjusting CV code, what do you think @heekinho @campos537 ?

Reply:
I think this changes are pretty useful! because depending on the application in the way it's done could cause some issues and every time that we create other application using it becomes harder to track all the changes. Also i agree on tracking by my side the changes that this changes on protocol-buffers will cause on the CV code

Reply:
Yep, we should go for it.